src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (x) => {
+    if (x !== undefined) {
```

Comment:
undefined will convert to false )

```suggestion
    if (x) {
```